static/src/javascripts/projects/commercial/modules/dfp/render-advert-label.js

```diff
@@ -10,12 +10,31 @@ const shouldRenderLabel = adSlotNode =>
         adSlotNode.getElementsByClassName('ad-slot__label').length
     );
 
+const createAdCloseDiv = (): HTMLDivElement => {
+    const closeDiv: HTMLDivElement = document.createElement('div');
+    closeDiv.className = 'ad-slot__closer';
+    closeDiv.innerHTML =
+        '<svg viewbox="0 0 40 40"><path class="ad-slot__closer__x" d="M 10,10 L 30,30 M 30,10 L 10,30" /></svg>';
+    closeDiv.onclick = function onclickMobileStickyCloser() {
+        this.closest('.mobilesticky-container').remove();
```

Comment:
I'm surprised `flow` didn't pick up but it's possible `this.closest('.mobilesticky-container')` will return `null`, if this happens you'll get an error `Cannot read property 'remove' of null`, I'd therefore suggest  assigning `this.closest('.mobilesticky-container')` to a variable `container` (or something like that)  and checking it exists before removing it.

```js
const container = this.closest('.mobilesticky-container');

if (container) {
  container.remove();
}
```

Reply:
👍